### ISSUE TITLE
Script Errors with 6.1.7, bWarnScriptErrors Not Functioning Correctly

### ISSUE THREAD

**Issue by SkyNinja** [xNVSE/NVSE#52]
After installing xNVSE 6.1.7, I began experiencing script errors as I played, apparently even within FalloutNV.esm, which should of course be impossible.

![20210727031944_1](https://user-images.githubusercontent.com/4765557/127142401-6152460d-dc5e-475e-af08-d02677cc94a4.jpg)

![20210727054826_1](https://user-images.githubusercontent.com/4765557/127142413-99abd506-ce68-495e-b4cd-25030548b024.jpg)

I don't know what triggered the first one, but the second was easily reproducible. I simply had to open the Pip-Boy and play around for a while. After testing several times, the script error occurred sooner or later every time while 6.1.7 was installed, but never occurred after switching back to 6.1.6, even after testing for much longer than it ever took for the error to occur under 6.1.7.

**Comment by c6-dev** [comment]
Add `bWarnScriptErrors=0` under `[Release]` in **nvse_config.ini**.

**Comment by SkyNinja** [comment]
> Only issue script error warnings in top corner if GECK is installed

Ah, okay, so these are things that always happened, they're just getting logged by default now because I have the Creation Kit installed, right?

**Comment by c6-dev** [comment]
Yep. Not a good idea in my opinion since it comes preinstalled with the GOG release.

**Comment by SkyNinja** [comment]
Inclined to agree, especially considering I've never seen the variable documented before the changelog, which can really add to the confusion.

**Comment by SkyNinja** [comment]
So, as it turns out, `bWarnScriptErrors` doesn't seem to work properly in 6.1.7. If it's 0 while the GECK is installed, script error warnings are still dumped to the console (not sure if this happened before, but I never saw it, despite regularly using the console and playing with Vanilla UI Plus since the beginning of this playthrough), and a blank message occasionally appears in the top corner, with no text and the default Vault Boy image.

**Comment by SkyNinja** [comment]
So, after further testing, I couldn't reproduce those script errors in the second screenshot at all under 6.1.6 with `bWarnScriptErrors=1`.

**Comment by c6-dev** [comment]
The option didn't exist in 6.1.6.
Are you sure that the setting is in the right section, the config file itself is in Data\NVSE, and not being overridden by a mod?

**Comment by SkyNinja** [comment]
> The option didn't exist in 6.1.6.

Ah, okay. I was thinking that [this](https://i.imgur.com/txHMXQ2.png) from the 6.0 changelog was referring to the same thing.

>Are you sure that the setting is in the right section, the config file itself is in Data\NVSE, and not being overridden by a mod?TTW is known to overwrite the config, for one.
Script errors don't appear in console on my end with the setting at 0.

Pretty sure, though if I did make a mistake that I'm missing, this screenshot should include it. I just ran the game with this config exactly as shown, and received script error logs for VUI+ printed to the console, though I didn't get the broken corner notification this time.
![image_2021-07-28_072140](https://user-images.githubusercontent.com/4765557/127321340-4229d84e-d595-43f2-81d0-85f2bad0205f.png)

**Comment by SkyNinja** [comment]
So, I recorded a video of this happening, to try and help.

https://youtu.be/9uISnk6uS0M

In the process, though, I may have made an important discovery: The errors only start getting dumped to the console after interacting with something, in this case, the door.

**Comment by Tommack63** [comment]
I've been getting those same errors, particularly for any script utilizing the removeMe function? Which  I thought as strange.

**Comment by korri123** [comment]
Will be fixed in 6.1.8

**Comment by SkyNinja** [comment]
Yep, behavior seems fixed. No logs unless `bWarnScriptErrors=1` or the GECK is running.

**Comment by SkyNinja** [comment]
Spoke too soon, same issues are still present, the error-that-isn't-actually-an-error in VUI+ simple isn't being mis-detected anymore. Now, it's logging an error in JIP CCC.

I get the corner notification and console log without the GECK running and with `bWarnScriptErrors` not present in the ini, and I get only console logs if `bWarnScriptErrors=0` is present.

**Comment by korri123** [comment]
Error in CCC is an actual error that wasn't logged before 6.1.8 (it was previously silent). By default, `bWarnScriptErrors` is true if the GECK was opened in the last two days.

**Comment by Tommack63** [comment]
> Spoke too soon, same issues are still present, the error-that-isn't-actually-an-error in VUI+ simply isn't being mis-detected anymore. Now, it's logging an error in JIP CCC.
> 
> I get the corner notification and console log without the GECK running and with `bWarnScriptErrors` not present in the ini, and I get only console logs if `bWarnScriptErrors=0` is present.

I get it regardless of bWarnScriptErrors value, and that due to I open geck quite often. seems to me we'll get this warning if we remove mods and then load a save too...  it could be the simplest of mods that just adds forms like items that you removed.

**Comment by Manan6619** [comment]
I can corroborate that the RemoveMe function is throwing these error messages still in 6.2.4. The script itself is working fine, I can see that the reference it enables is being enabled and the item the script is on is not present in my inventory.

**Comment by Demorome** [comment]
There was a bug related to how `bWarnScriptErrors` was being read which was fixed [here](https://github.com/xNVSE/NVSE/pull/86) . The fix is included in 6.2.5. Could you confirm if the issue persists?
